sss/tests/test_sss.py

```diff
@@ -18,6 +22,25 @@ def test_read_file(setup_and_teardown_package):
 
     #session = setup_and_teardown_package.sessionmaker()
 
+def test_check_extra_columns(setup_and_teardown_package):
+    """ Test checking the extra columns: miscellaneous_is_secondary, health_care_is_secondary, analysis_is_secondary """
+
+    df, file = sss_table.read_file(os.path.join(DATA_PATH, 'AR2022_SSS_Full.xlsx'))
+    df, miscellaneous, health_care, arpa = sss_table.check_extra_columns(df)
+
+    assert 'miscellaneous_is_secondary' and 'health_care_is_secondary' and 'analysis_is_secondary' in df.columns
+
+def test_check_extra_columns_error(setup_and_teardown_package):
+    with pytest.raises(ValueError, match="df should be a pandas dataframe."):
+        df, miscellaneous, health_care, arpa = sss_table.check_extra_columns((os.path.join(DATA_PATH)), 'AR2022_SSS_Full.xlsx')
```

Comment:
This test doesn't pass on my machine. I think there's a problem with how the parentheses are arranged, you need to be joining `DATA_PATH` with `'AR2022_SSS_Full.xlsx'` before passing it to the function.